src/core_plugins/metrics/public/components/modal_confirm.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {
+  EuiConfirmModal as ConfirmModal,
+  EuiOverlayMask as ModalOverlay,
+  EUI_MODAL_CONFIRM_BUTTON as MODAL_CONFIRM_BUTTON
+} from '@elastic/eui';
+export function ModalConfirm({
+  title = 'Are you sure?',
```

Comment:
The EUI writing guidelines suggest to `Avoid using "Are you sure"` and labeling the buttons with their action, e.g. `Switch` and `Keep`.

Reply:
This is just the component. What's better then `Are you sure?` for a default confirmation? I'm not going to change `Yes` and `No` for the defaults; when using this component, future developers can override the defaults with better syntax (as I did when I used it).

Reply:
Yes, I can't think of better defaults myself TBH. Maybe not having default values at all in a generic component might be appropriate?

Reply:
OR put something like `title='TITLE GOES HERE'` and then just leave the buttons as `Yes` and `No`.

Reply:
@simianhacker what if you use prop-types and make it a required prop?

Reply:
@mattapperson We could do that too, let's be honest though... it's probably only ever going to be used in this instance. We've probably already talked too much about this, the end results has the correct text and this component only exists because I wrote it using KUI knowing that eventually I would have to swap it out with EUI and I just wanted to do that in one place.